android/guava-tests/test/com/google/common/base/ObjectsTest.java

```diff
@@ -46,7 +46,7 @@ public void testEqual() throws Exception {
 
     // test distinct string objects
     String s1 = "foobar";
-    String s2 = new String(s1);
+    String s2 = s1;
     assertTrue(Objects.equal(s1, s2));
```

Comment:
Here and in some test below distinct _instances_ are created on purpose. These changes should be reverted.

Reply:
thx, reverted assuming it as SSOT and is replicated somehow:

<img width="1383" height="520" alt="image" src="https://github.com/user-attachments/assets/7c23570f-704b-44be-a33f-9757f7db7dcb" />